src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -484,88 +499,91 @@ namespace xdp {
       // 1. Reserve and start core module counters
       //
       int numCoreCounters = 0;
-      XAie_ModuleType mod = XAIE_CORE_MOD;
-      for (int i=0; i < coreCounterStartEvents.size(); ++i) {
-        auto perfCounter = core.perfCounter();
-        auto ret = perfCounter->initialize(mod, coreCounterStartEvents.at(i),
-                                           mod, coreCounterEndEvents.at(i));
-        if (ret != XAIE_OK) break;
-        ret = perfCounter->reserve();
-        if (ret != XAIE_OK) break;
+      if (useCounters) {
+        XAie_ModuleType mod = XAIE_CORE_MOD;
+        for (int i=0; i < coreCounterStartEvents.size(); ++i) {
+          auto perfCounter = core.perfCounter();
+          auto ret = perfCounter->initialize(mod, coreCounterStartEvents.at(i),
+                                            mod, coreCounterEndEvents.at(i));
+          if (ret != XAIE_OK) break;
+          ret = perfCounter->reserve();
+          if (ret != XAIE_OK) break;
 
-        // NOTE: store events for later use in trace
-        XAie_Events counterEvent;
-        perfCounter->getCounterEvent(mod, counterEvent);
-        int idx = static_cast<int>(counterEvent) - static_cast<int>(XAIE_EVENT_PERF_CNT_0_CORE);
-        perfCounter->changeThreshold(coreCounterEventValues.at(i));
+          // NOTE: store events for later use in trace
```

Comment:
Hmm...looks like a tab was added.  Please use only spaces for intention.

Reply:
removed tab